codetransformer/instructions.py

```diff
@@ -0,0 +1,163 @@
+from abc import ABCMeta, abstractmethod
+from dis import opname, opmap, hasjabs, hasjrel, HAVE_ARGUMENT, stack_effect
```

Comment:
See comments elsewhere regarding my feelings about formatting of lots of imports...